vision/product_search/product_search_add_product_to_product_set.rb

```diff
@@ -15,12 +15,12 @@
 # [START vision_product_search_add_product_to_product_set]
 require "google/cloud/vision"
 
-def product_search_add_product_to_product_set(
+def product_search_add_product_to_product_set \
```

Comment:
I found this change to be a little confusing when compared to other samples. I understand the usage, but I would find using parenthesis here easier to parse. (opinion).

Reply:
hard agree

Reply:
whoops, didn't realize I had an outdated section open, plz to disregard